aospy/internal_names.py

```diff
@@ -61,7 +61,7 @@
      (PHALF_STR, ('phalf',)),
      (PFULL_STR, ('pfull',)),
      (PLEVEL_STR, ('level', 'lev', 'plev')),
-     (TIME_STR, ('time', 'XTIME')),
+     (TIME_STR, ('time', 'XTIME', 'T')),
```

Comment:
I'll defer to @spencerahill's judgement on this one, but I'll note that `'T'` is also used to represent temperature in the WRF model, so we might hold off on adding this as an automatic alternative name for `'time'`.  I've fast-tracked an implementation of #182 (see #297), which might be the preferred solution in this particular case.

Reply:
Thanks @spencerkclark, I agree that with #297 now in the pipeline, let's limit this PR to 'X' and 'Y', since we can be pretty sure that those won't cause any problems in the way that 'T' might.

@jdossgollin, does that work for you?  Thanks much for the PR!

Reply:
Yes works for me -- in discussion of #293 I suggested that a `rename` in the `preprocess` function could also take care of something like time as `T`.

I'm sorry to say I'm not sure what I should do to finish patching this PR through -- do I edit the code in here and commit back in? Thanks for the patience!

Reply:
On your local machine, make sure you're still on this branch, make your edits, `git commit` them, and then once you're ready `git push` the branch again to here.  The PR will be updated accordingly.

Please also add an entry to our [What's New](https://github.com/spencerahill/aospy/blob/develop/docs/whats-new.rst) docs page.  Just mimic the structure of the other entries, under the v0.3 heading (don't worry about the v0.3 vs. v0.2.1 bit; we'll fix that later).

Finally, for almost all changes or additions to the codebase, we require formal tests that ensure that the new code actually works as we intended and didn't break anything unintentionally.  These are in the [test](https://github.com/spencerahill/aospy/tree/develop/aospy/test) directory.  However, this may be one of the rare exception where we're OK without.  I don't think we have any existing tests specifically of the different options in `internal_names`, so it would be a lot to ask of you to create it.  @spencerkclark what do you think?

Reply:
I agree; I think we're OK without tests in this case.  The grid attribute renaming method is exercised by the tests [here](https://github.com/spencerahill/aospy/blob/a2cd51fc1cb33c0667e02b6d3a14ac529784f553/aospy/test/test_data_loader.py#L155-L192).  If things work for one alternate name/internal name pair (in the case of the tests `'LATITUDE'`->`'lat'`) then I would expect things should work for another pair just as well.

Reply:
@jdossgollin a nice (fairly comprehensive) resource for learning about Git/GitHub/testing/documentation in the context of contributing to open source Python projects is the [xarray contributing guide](http://xarray.pydata.org/en/stable/contributing.html).  In an abstract sense (i.e. replace xarray with aospy) much of what is in there applies here.